docs/components/component/_index.md

```diff
@@ -27,45 +27,124 @@ Most robots with a COMPONENT need at least the following hardware:
 
 ## Configuration
 
-Configuring this component on your robot enables you to ....
-You can easily configure your component, as pictured below, on the [Viam app](https://app.viam.com/).
-{{< tabs name="TabPanelExample" >}}
-{{% tab name="UI"%}}
+For configuration information, click on your COMPONENT's model:
```

Comment:
I like this but I also think that `supported models include` is worthwhile, do you have any ideas for an in-between/combo that wouldn't make it look extra wordy? I don't really 😆 other than that LGTM

Reply:
Yeah I agree. Hmm. Maybe this?
```suggestion
Find configuration information for each supported model at the links below:
```

Reply:
How about: 
```suggestion
The COMPONENT component supports the following models. Click on the component model name for detailed configuration information. 
```

Reply:
```suggestion
The COMPONENT component supports the following models.
Click on the component model name for detailed configuration information.
```
I like that! Just adding a newline and removing a trailing space. Do we throw a colon on the end or no?

Reply:
Either works so go for whichever you prefer :)